homeassistant/components/freedompro/light.py

```diff
@@ -43,6 +43,14 @@ def __init__(self, hass, api_key, device, coordinator):
         self._attr_unique_id = device["uid"]
         self._type = device["type"]
         self._characteristics = device["characteristics"]
+        self._attr_device_info = {
+            "name": self.name,
+            "identifiers": {
+                (DOMAIN, self.unique_id),
```

Comment:
I'm a tiny bit confused now though, devices & entities are not the same things in Home Assistant, yet, the entity unique ID is used as device identifier?

Reply:
Yes, in reality, each device is composed of a single entity, this was done to be able to call up the device for example in an action automation

Reply:
hi @frenck, the change you suggested to me, if you confirm that it's okay, I also apply it to all the other pull requests

Thank you

Reply:
It's fine if the UID which you use as unique_id for the light identifies a physical devices, like a bulb, and not just a light endpoint of a bulb.

Reply:
the UID identify a physical device